lib_board_support/src/boards/xk_audio_316_mc_ab/xk_audio_316_mc_ab_board.xc

```diff
@@ -38,14 +38,26 @@ on tile[0]: port p_margin = XS1_PORT_1G;     /* CORE_POWER_MARGIN:   Driven 0:
                                               *                      Pull-up:    0.854v
                                               *                      Driven 1:   0.85v
                                               */
+
+void xk_audio_316_mc_ab_core_voltage_reduce(const int reduce_voltage)
```

Comment:
Prefer "core_voltage_set" or similar (as half the time you're increasing the voltage ;) )